Sources/XCHammer/XcodeTarget.swift

```diff
@@ -536,6 +536,8 @@ public class XcodeTarget: Hashable, Equatable {
                 break // These attrs are not related to XCConfigs
             case .binary:
                 break // Explicitly not handled since it is a implicit target we don't intend to handle
+            case .has_swift_info, .has_swift_dependency:
+                break
```

Comment:
I think we need to actually implement something here to populate the correct version of Swift for this target. You'll notice Tulsi does something similar
```
      for entry in targetRules {
        if let swiftVersion = entry.attributes[.swift_language_version] as? String {
          buildSettings["SWIFT_VERSION"] = swiftVersion
        } else if entry.attributes[.has_swift_dependency] as? Bool ?? false {
          buildSettings["SWIFT_VERSION"] = fetchDefaultSwitchVersion()
        }
        if let swiftToolchain = entry.attributes[.swift_toolchain] as? String {
          buildSettings["TOOLCHAINS"] = swiftToolchain
        }
      }
```

I'm assuming setting the `SWIFT_VERSION` and adding another property to `XCBuildSettings` is all that we need to do